--- caffe-1.0.orig/include/caffe/common.hpp	2017-04-15 13:17:48.000000000 -0300
+++ caffe-1.0/include/caffe/common.hpp	2019-06-23 12:29:11.268673103 -0300
@@ -69,6 +69,12 @@
 // is executed we will see a fatal log.
 #define NOT_IMPLEMENTED LOG(FATAL) << "Not Implemented Yet"
 
+// Supporting OpenCV4
+#if (CV_MAJOR_VERSION == 4)
+#define CV_LOAD_IMAGE_COLOR cv::IMREAD_COLOR
+#define CV_LOAD_IMAGE_GRAYSCALE cv::IMREAD_GRAYSCALE
+#endif
+
 // See PR #1236
 namespace cv { class Mat; }
 
