diff --git a/CMakeLists.txt b/CMakeLists.txt
index 27d172f9..50fd448c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -101,11 +101,6 @@ add_subdirectory(docs)
 # ---[ Linter target
 add_custom_target(lint COMMAND ${CMAKE_COMMAND} -P ${PROJECT_SOURCE_DIR}/cmake/lint.cmake)
 
-# ---[ pytest target
-if(BUILD_python)
-  add_custom_target(pytest COMMAND python${python_version} -m unittest discover -s caffe/test WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/python )
-  add_dependencies(pytest pycaffe)
-endif()
 
 # ---[ uninstall target
 configure_file(
